app/models/friendship.rb

```diff
@@ -0,0 +1,67 @@
+class Friendship < ActiveRecord::Base
+  extend TokenContext
+  include Tokenizer
+	enum friendship_status: [ :denied, :pending, :accepted]
+  attr_accessor :friendship_request_token
+	cattr_accessor :daily_request_limit
```

Comment:
Oh, I learned something then.  I didn't realize `cattr_accessor` was a thing.  I assumed you meant to write `attr_accessor`.